src/test/java/stringcalculator/CalculatorNumber.java

```diff
@@ -0,0 +1,44 @@
+package stringcalculator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class CalculatorNumber {
+    private static final Map<Integer, CalculatorNumber> cache = new HashMap<>();
```

Comment:
```suggestion
public class CalculatorNumber {
    private static final Map<Integer, CalculatorNumber> CACHE = new HashMap<>();
```

자바 네이밍 규칙을 지켜주면 좋을 것 같아요. :)